### PR TITLE
Begin to update naming information

### PR DESCRIPTION
**Pull request by PolarBean**
Here we begin to update the name of the tool so that we begin to standardise with other brainglobe tools.

### CODE REVIEW

**Comment by PolarBean** [comment]
Just the renaming changes we discussed. Let me know if you think this looks good!

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
-# CCF_translator
+# brainglobe-ccf-translator
 ![PyPI - Version](https://img.shields.io/pypi/v/CCF-translator)
```
Comment (adamltyson) [inline README.md:2]:
```suggestion
![PyPI - Version](https://img.shields.io/pypi/v/brainglobe-ccf-translator)
```

**Review by adamltyson — APPROVED** [review]
Looks good to me. I made one suggestion, as I noticed the PyPI version badge was now out of date.

File: README.md (hunk 1 above)
Comment (PolarBean) [reply README.md]: good catch!

**Comment by PolarBean** [comment]
Great thanks for finding that. Ill merge this now

**Pull request merged**